### PR TITLE
Bump config to 1.3.3

### PR DESCRIPTION
**Pull request by naferx**

### CODE REVIEW

**Review by raboof — APPROVED** [review]

File: project/Dependencies.scala [hunk 1]
```diff
@@ -44,7 +44,7 @@ object Dependencies {
     val camelCore = "org.apache.camel" % "camel-core" % "2.17.7" exclude ("org.slf4j", "slf4j-api") // ApacheV2
 
     // when updating config version, update links ActorSystem ScalaDoc to link to the updated version
-    val config = "com.typesafe" % "config" % "1.3.2" // ApacheV2
+    val config = "com.typesafe" % "config" % "1.3.3" // ApacheV2
```
Comment (patriknw) [inline project/Dependencies.scala]: I assume that means it is binary backwards compatible

**Review by patriknw — APPROVED** [review]
LGTM

File: project/Dependencies.scala (hunk 1 above)
Comment (naferx) [reply project/Dependencies.scala]: yeah, I guess

Comment (ktoso) [reply project/Dependencies.scala]: Yes it is :)

**Pull request merged**